pkg/morph/client/netmap/netmap.go

```diff
@@ -36,7 +36,45 @@ type EpochSnapshotValues struct {
 	*GetNetMapValues
 }
 
-const nodeInfoFixedPrmNumber = 1
+// GetNetMapCandidatesArgs groups the arguments
+// of get network map candidates test invoke call.
+type GetNetMapCandidatesArgs struct {
+}
+
+// GetNetMapCandidatesValues groups the stack parameters
+// returned by get network map candidates test invoke.
+type GetNetMapCandidatesValues struct {
+	NetmapNodes []*PeerWithState
```

Comment:
Pls make it unexported, or it will be hard to keep code and logic compatibility.

Reply:
made unexported and add getter to it